core/src/main/resources/hudson/cli/Messages_fr.properties

```
@@ -0,0 +1,101 @@
+ Liste tous les jobs pour une vue spécifique ou un groupe spécifique d''items.
+ListPluginsCommand.ShortDescription=\
+ Émet une liste de plugins installés.
+SetBuildDescriptionCommand.ShortDescription=\
+ Définit la description d''un build.
+RemoveJobFromViewCommand.ShortDescription=\
+ Supprime des jobs d''une vue.
+VersionCommand.ShortDescription=\
+ Émet la version courante.
+GetJobCommand.ShortDescription=\
+ Exporte la définition XML du job dans la sortie standard.
+GetNodeCommand.ShortDescription=\
+ Exporte la définition du nœud dans la sortie standard.
+GetViewCommand.ShortDescription=\
+ Exporte la définition XML de la vue dans la sortie standard.
+SetBuildDisplayNameCommand.ShortDescription=\
+ Définit le nom d''afichage d''un build.
+WhoAmICommand.ShortDescription=\
+ Génère un rapport de vos informations d''identification et de vos droits.
+UpdateJobCommand.ShortDescription=\
+ Met à jour la définition XML du job depuis l''entrée standard. Opposé de la commande "get-job".
+UpdateNodeCommand.ShortDescription=\
+ Met à jour la définition XML du nœud depuis l''entrée standard. Opposé de la commande"get-node".
+SessionIdCommand.ShortDescription=Exporte l''identifiant de session, qui est modifié à chaque redémarrage de Jenkins.
+UpdateViewCommand.ShortDescription=\
+ Met à jour la définition XML de la vue depuis l''entrée standard. Opposé de la commande "get-view".
+
+BuildCommand.CLICause.ShortDescription=Démarré depuis la ligne de commande par {0}
+BuildCommand.CLICause.CannotBuildDisabled=\
+Impossible de construire le build {0} car il est désactivé.
+BuildCommand.CLICause.CannotBuildConfigNotSaved=\
+Impossible de construire le build {0} car sa configuration n''a pas été sauvegardée.
+BuildCommand.CLICause.CannotBuildUnknownReasons=\
+Impossible de construire le build {0} pour des raisons inconnues.
+
+DeleteNodeCommand.ShortDescription=Supprime un/des nœud(s)
+ReloadJobCommand.ShortDescription=Recharge un/des job(s)
+OnlineNodeCommand.ShortDescription=Reprise avec un nœud dédié aux builds, afin d''annuler la commande "offline-node" précédente.
+ClearQueueCommand.ShortDescription=Efface la file d''attente de build.
+ReloadConfigurationCommand.ShortDescription=Efface toutes les données chargées en mémoire et recharge l''intégralité depuis les fichiers système. Utile quand vous avez modifié des fichiers de configuration directement sur disque.
+ConnectNodeCommand.ShortDescription=Se reconnecte à un nœud.
+DisconnectNodeCommand.ShortDescription=Se déconnecte d''un nœud.
+QuietDownCommand.ShortDescription=Stoppe Jenkins silencieusement en préparation d''un redémarrage. Ne démarre aucun build.
+SafeRestartCommand.ShortDescription=Redémarre Jenkins en sûreté. Ne démarre aucun build.
+CancelQuietDownCommand.ShortDescription=Annule l''effet de la commande "quiet-down".
+OfflineNodeCommand.ShortDescription=Arrête temporairement d''utiliser un nœud dédié aux builds, jusqu''à la prochaine commande "online-node".
```

```suggestion
OfflineNodeCommand.ShortDescription=Arrête temporairement d''utiliser un nœud dédié aux builds jusqu''à la prochaine commande "online-node".
```
Comma not useful in French.